plugins/mail-job/impl/pom.xml

```diff
@@ -55,5 +55,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-platform-extensions</artifactId>
```

Comment:
Same comment as the first one; this dependency cannot be here unless this plugin moves to the highdeps section.